source/_posts/2018-07-11-hassio-images.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "New Hass.io images"
+description: "We release our new images."
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 months we are happy to announce our new images for Hass.io, based on HassOS.
+
+Note that the descriptions of the new system can get quite technical. Feel free to jump to the [migration steps].
+
+## {% linkable_title What is new %}
+
+We have build HassOS on top of the [Buildroot] framework. The focus of the system is to be a very small and high efficient operating system to run Docker like a hypervisor. It has installed just enough software to run a supervisor. We have also focussed on security, there are, for example, no default passwords and we use [AppArmor] to protect the applications and containers on HassOS.
+
+Key features:
+
+- Safe and secure updating with [Rauc] over USB or internet (OTA)
+- Uses an LZ4 compressed root filesystem and parts of the memory
+- Read-only root filesystem, designed to run on embedded systems
+- Dbus connected hosts services
+- Latest LT Linux kernel
+- Latest Docker-ce version
+- Fully supported NetworkManager
+- Bluetooth support using Bluez
+- Supports lots different Hardware
+
+## {% linkable_title Migration %}
+
+The design of HassOS is different from ResinOS. Because of this, we can't provide an OTA update from the old (ResinOS) too new system (HassOS).
+
+You need to perform the following steps to upgrade:
+
+1. If you have installed the Bluetooth add-on, please remove it, since it is not required anymore.
+2. Make a Hass.io snapshot of your current system and download it to your computer.
+3. Download the latest [HassOS stable] version.
+4. Flash the downloaded HassOS image with [Etcher] to your SD card.
+5. Raspberry Pi: In case you have modified the `config.txt` (in the boot partition), you will also need to apply these changes to HassOS. Do **NOT** simply copy from the file from your old setup into HassOS! Apply those changes manually!
```

Comment:
How can a user apply those changes? Link to instructions ?

Reply:
Agree, unclear to me as well.
Those are actually kinda undocumented changes (maybe they are documented at the Raspberry Pi website?).

Nevertheless, a user who has modified this is most likely well aware.